rotkehlchen/tests/unit/test_tokens.py

```diff
@@ -473,6 +473,32 @@ def test_erc721_token_ownership_verification(
         token_kind=EvmTokenKind.ERC721,
         collectible_id='7809',
     )
+    # regression test: broken erc721 implementations to ensure token detection
+    # doesn't fail when ownerOf calls return empty or fail
+    broken_erc721_token_1 = get_or_create_evm_token(
+        userdb=ethereum_inquirer.database,
+        evm_inquirer=ethereum_inquirer,
+        evm_address=string_to_evm_address('0x0E3A2A1f2146d86A604adc220b4967A898D7Fe07'),
+        chain_id=ChainID.ETHEREUM,
+        token_kind=EvmTokenKind.ERC721,
+        collectible_id='1',
+    )
+    broken_erc721_token_2 = get_or_create_evm_token(
+        userdb=ethereum_inquirer.database,
+        evm_inquirer=ethereum_inquirer,
+        evm_address=string_to_evm_address('0x6EbeAf8e8E946F0716E6533A6f2cefc83f60e8Ab'),
+        chain_id=ChainID.ETHEREUM,
+        token_kind=EvmTokenKind.ERC721,
+        collectible_id='2',
+    )
+    broken_erc721_token_3 = get_or_create_evm_token(
+        userdb=ethereum_inquirer.database,
+        evm_inquirer=ethereum_inquirer,
+        evm_address=string_to_evm_address('0xFaC7BEA255a6990f749363002136aF6556b31e04'),
+        chain_id=ChainID.ETHEREUM,
+        token_kind=EvmTokenKind.ERC721,
+        collectible_id='3',
+    )
```

Comment:
why add 3 times the same case? Do all of them fail in the same way? By your comment I beliefe you only need two of them

Reply:
indeed only two are needed. i just copy pasted the arguments from the logs.